extensions/brb.md

```diff
@@ -0,0 +1,93 @@
+---
+title: IRCv3 `brb` Extension
+layout: spec
+extends: resume
+copyrights:
+  -
+    name: "Jess"
+    period: "2019"
+    email: "jess@jesopo.uk"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `brb` capability name. Instead, implementations SHOULD
+use the `draft/brb` capability name to be interoperable with other
+software implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed capability name.
+
+## Introduction
+
+Occasionally, clients will need to restart in order to update software and, at
+the moment, this will cause a `QUIT` to be sent out for the user on all their 
+networks (unless the client supports in-place upgrading e.g. weechat.)
```

Comment:
We should avoid mentioning software names explicitly, just leave it at "...supports upgrading in-place)"